protocol/app/app.go

```diff
@@ -1524,6 +1554,7 @@ func initParamsKeeper(
 	paramsKeeper.Subspace(govtypes.ModuleName).WithKeyTable(govv1.ParamKeyTable()) //nolint:staticcheck
 	paramsKeeper.Subspace(crisistypes.ModuleName)
 	paramsKeeper.Subspace(ibctransfertypes.ModuleName)
+	paramsKeeper.Subspace(icahosttypes.SubModuleName)
```

Comment:
I believe not not since `params` state is empty (see sample pregenesis)

Reply:
Great question. Params module is used in [ibc-go V7 but deprecated in V8](https://github.com/cosmos/ibc-go/blob/66532056e85e7184a8edb48b2a46d445c172ac36/docs/docs/05-migrations/11-v7-to-v8.md#params-migration). IIUC we will do the IBC v8 migration in rc0, so later we will just remove this. 

Even if we continue to use v7, I don't think any migration is needed since `.Subspace` doesn't update any state and the `paramSpace` is set when `icaAppModule` is initialized.

Reply:
Okay, I agree as long as this doesn't require any state changes